lib/python/satos_payload_sdk/__init__.py

```diff
@@ -0,0 +1,7 @@
+import logging
+
+if __name__ == '__main__':
```

Comment:
I'm not sure what this file is required for. Can you explain?

Reply:
It is used for logging satOS SDK , just in case it was called by itself and not from payload app.
@bcwaldon  Should I delete it?

Reply:
SDK is meant to be used by some application. It won't become application itself. I don't think this file is required.

Reply:
OK, change done